code_producers/src/llvm_elements/array_switch.rs

```diff
@@ -36,7 +41,7 @@ mod array_switch {
         let bigint_ty = bigint_type(producer);
         let i32_ty = i32_type(producer);
         let args = &[
-            bigint_ty.ptr_type(Default::default()).into(),
+            bigint_ty.array_type(0).ptr_type(Default::default()).into(),
```

Comment:
Yeah, I think so. Something similar to the `*_run` functions

Reply:
Yeah that's what these will generate now (which is nice for uniformity).